servers/parser-graphql-wrapper/src/config/index.ts

```diff
@@ -58,4 +58,115 @@ export default {
   },
   parserEndpoint: process.env.PARSER_URL || 'http://example-parser.com/',
   parserRetries: 3,
+  intMask: {
```

Comment:
you don't have to do this now, but it's probably worth exporting some kind of config or default setters from the library